docs/05-concepts/06-database/01-connection.md

```diff
@@ -4,9 +4,14 @@ In Serverpod the connection details and password for the database are stored ins
 
 Serverpod automatically establishes a connection to the Postgresql instance by using these configuration details when you start the server.
 
+
+## Connecting to a Postgresql Docker Image
```

Comment:
Suggestion: Remove this header.

The details in this section are general for all connection types regardless of whether we are connecting to a Postgres instance in a docker container or if it is running somewhere else.

The information about how to connect to the Postgresql docker image is outlined in the `Development database` header. 

This section should still be kept general.